web/test/pages/dashboard.test.tsx

```diff
@@ -668,33 +668,173 @@ describe("dashboard page", () => {
       }
     );
 
-    it.each(operatingPhasesWithQuickActions)(
-      "displays state-contracting-external-link quick action when industry id is not home-contractor for %s",
-      (phase) => {
-        useMockProfileData({ operatingPhase: phase, industryId: "some-value" });
-        renderDashboardPage({
-          quickActionLinks: [
-            generateQuickActionLink({
-              name: "some name",
-              filename: "state-contracting-external-link",
-              externalRoute: "some-external-route",
-            }),
-          ],
-        });
-        expect(screen.getByText("some name")).toBeInTheDocument();
-      }
-    );
+    describe("state-contracting-external-link", () => {
+      it.each(operatingPhasesWithQuickActions)(
+        "displays state-contracting-external-link quick action when business persona is oscar and sector id is not construction for %s",
+        (phase) => {
+          useMockProfileData({ operatingPhase: phase, sectorId: "some-value", businessPersona: "OWNING" });
+          renderDashboardPage({
+            quickActionLinks: [
+              generateQuickActionLink({
```

Comment:
[pebble / sand] I know you just followed the existing pattern but do you there is value in making `some name` into something that is a little more descriptive of what we are expecting in the document just because `some name` sounds random but here we are expecting the things to be related to the state-contracting-external-link quick action? 

```
generateQuickActionLink({
  name: "state-contracting-quick-action",
  filename: "state-contracting-external-link",
  externalRoute: "state-contracting-external-route",
}),

```

Reply:
I think this thought is applicable to tests below as well.